src/claim/unique-id/const-object-types.js

```diff
@@ -0,0 +1,10 @@
+const CONST_TYPE_OBJECT = {
```

Comment:
I find this name not very explanatory.  Could we use another name like... ID_TYPE? or PERSONAL_ID_TYPE?

Reply:
removed `CONST_` since SCREAMING_SNAKE_CASE denotes constants variables